src/classes/Commands/Commands.ts

```diff
@@ -94,7 +94,8 @@ export default class Commands {
     }
 
     async processMessage(steamID: SteamID, message: string): Promise<void> {
-        const command = CommandParser.getCommand(message.toLowerCase());
+        const customSteamPrefix = this.bot.options.miscSettings?.prefixes?.steam ?? '!';
```

Comment:
As far as I am aware, the Discord UI also use this `processMessage` method.
Maybe use `this.bot.getPrefix(steamID)` instead of this?

Reply:
Fixed